docs/GettingStarted.md

```diff
@@ -945,7 +945,7 @@ connection = Excon.new('https://example.com')
 connection.get
 ```
 
-Where `options` is an optional `Hash` that accepts the following parameters:
+`options` are the following keyword arguments:
 
 | Key | Description | Default |
 | --- | ----------- | ------- |
```

Comment:
⬇️ I noticed on line 965 and 974 we're not using `**options`. May be worth checking if those also need to be updated.

Reply:
I think they are not keyword arguments
```
def self.with(options = {})
```